### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by iAnonymous3000**
Update privacy policy

- Remove newsletter and tracking mentions
- Add RSS feed approach
- Add technical infrastructure details
- Update communication channels
- Enhance privacy commitments

### CODE REVIEW

File: content/privacy-policy/index.md [hunk 1]
```diff
@@ -1,29 +1,95 @@
...
+- **No Personal Data Collection**: Our static website does not collect or store personal information
+- **No Cookies**: We do not use cookies or tracking mechanisms
+- **RSS Feeds**: We use RSS feeds for updates instead of email subscriptions, ensuring user privacy
+- **Cloudflare Analytics**: Limited anonymous metrics may be collected by Cloudflare Pages for operational purposes
+- **No Newsletter Data**: We do not collect email addresses or maintain subscriber lists
+
+## 2. Communication Channels
+
+When you interact with us through our official channels:
+- RSS Feeds for updates
```
Comment (Parthiv-M) [inline content/privacy-policy/index.md:30]: Can we add _work in progress_ here?

**Review by Parthiv-M — CHANGES_REQUESTED** [review]
LGTM, just the minor change

File: content/privacy-policy/index.md (hunk 1 above)
Comment (iAnonymous3000) [reply content/privacy-policy/index.md:30]: Sure yes! or shall we wait for RSS to be done by @Samyukta-14 ?

Comment (Parthiv-M) [reply content/privacy-policy/index.md:30]: Let's mark it as WIP, I don't think the issue is being worked on at the moment (#22)

**Pull request merged**